src/Magnum/Trade/Test/MaterialDataTest.cpp

```diff
@@ -179,23 +197,27 @@ void MaterialDataTest::constructMovePhong() {
 
     const int a{};
     PhongMaterialData data{
-        PhongMaterialData::Flag::AmbientTexture|PhongMaterialData::Flag::DiffuseTexture|PhongMaterialData::Flag::SpecularTexture|PhongMaterialData::Flag::NormalTexture|PhongMaterialData::Flag::TextureTransformation,
-        0x111111_rgbf, 1,
-        0xeebbff_rgbf, 42,
-        0xacabad_rgbf, 24, 17,
+        PhongMaterialData::Flag::AmbientTexture|PhongMaterialData::Flag::DiffuseTexture|PhongMaterialData::Flag::SpecularTexture|PhongMaterialData::Flag::NormalTexture|PhongMaterialData::Flag::TextureTransformation|PhongMaterialData::Flag::TextureCoordinateSets,
+        0x111111_rgbf, 1, 0,
+        0xeebbff_rgbf, 42, 1,
+        0xacabad_rgbf, 24, 2, 17, 3,
         Matrix3::rotation(90.0_degf),
         MaterialAlphaMode::Blend, 0.55f, 13.0f, &a};
```

Comment:
Okay but why is this verified only in a test for move?

For completeness the `constructPhongTextured()` should check that accessing those doesn't result in assert and returns 0, and there should be `constructPhongTexturedCoordinateSets()` that's basically a copy plus it also enables the flag and fills in the coordinate sets.